src/components/Explore/TimeSeriesChart.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+import {
+  LineChart,
+  Line,
+  CartesianGrid,
+  XAxis,
+  YAxis,
+  Tooltip,
+  Legend
+} from 'recharts';
+import { connect } from 'react-redux';
+import '../../styles/TimeSeries.css';
+
+const stroke = ['red', 'blue', 'green', 'purple', 'orange', 'grey', 'yellow'];
+
+class TimeSeries extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    console.log('In Render');
```

Comment:
Can you remove `console.log` statements? These preferably should only be in your on development branch, if each of us left a few of these around, it will get chaotic real quick.

Reply:
I still need to implement the expand functionality, which is causing some problem. I will remove in the next push.